fintech-examples/fintech-api/pom.xml

```diff
@@ -0,0 +1,127 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>de.adorsys.opba</groupId>
+    <artifactId>fintech-examples</artifactId>
+    <version>0.0.4-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>fintech-api</artifactId>
+  <name>fintech-api</name>
+
+  <properties>
+    <!-- Hard reference to root directory. Used to reference pmd anc checkstyle files from root.
+    Change this if depth changes. Current: /open-banking-gateway/fintech-examples/fintech-api -->
+    <main.basedir>${project.parent.parent.basedir}</main.basedir>
+    <openapi-spec-file-location>${project.basedir}/src/main/resources/static/fintech_api.yml</openapi-spec-file-location>
+  </properties>
+
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-webmvc</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.swagger</groupId>
+      <artifactId>swagger-annotations</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>javax.validation</groupId>
+      <artifactId>validation-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+    </dependency>
+  </dependencies>
+  
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>io.swagger.codegen.v3</groupId>
+        <artifactId>swagger-codegen-maven-plugin</artifactId>
+        <version>${swagger-codegen-maven-plugin.version}</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+            <configuration>
+              <inputSpec>${openapi-spec-file-location}</inputSpec>
+              <language>spring</language>
+              <output>${project.build.directory}/generated-sources/open-api/</output>
+              <configOptions>
+                <sourceFolder>src/main</sourceFolder>
```

Comment:
please to src/main/java
(I allredy changed this in  banking-rest-api and consent-rest-api)